lib/mutant/mutator/node/arguments.rb

```diff
@@ -59,7 +55,7 @@ def emit_mlhs_expansion
           mlhs_childs_with_index.each do |child, index|
             dup_children = children.dup
             dup_children.delete_at(index)
-            dup_children.insert(index, *child.children)
+            dup_children.insert(index, *child)
```

Comment:
It does. It implements `#to_ary`. delegating to `#children`. I do not like it but given its supported API mutant forces the least powerful primitive in the current environment.